platform-tests/upstream/run_acceptance_tests.sh

```diff
@@ -7,6 +7,23 @@ if  [ "${DISABLE_CF_ACCEPTANCE_TESTS:-}" = "true" ]; then
   exit 0
 fi
 
+# FIXME: Remove this once we are deploying a version of cf-release
+# that includes capi-release >= 1.35.0.
+(
+  cd  "$(pwd)/cf-release/src/github.com/cloudfoundry/cf-acceptance-tests/"
```

Comment:
TBH I was sure that git resource has `version` filter so it's possible to select specific SHA but I can't find anything like this in docs 🤔 . It's a temp change so I guess I's good if it works.